lib/codes/itunes_connect.rb

```diff
@@ -41,6 +41,15 @@ def run(args)
 
       text_fields.first.set(number_of_codes.to_s)
       click_next
+
+      # are there any errors ?
+      errors = []
+      begin
+         errors = wait_for_elements("div[id=LCPurpleSoftwarePageWrapperErrorMessage]")
+      rescue Exception
+      end
+      raise errors.first.text.red unless errors.count == 0
```

Comment:
I believe we should be covered. The <div> is described here:

https://github.com/fastlane/codes/issues/16#issue-98120709

and encompasses all underlying errors. I can add HTML extract documentation to the code if you want.